src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/softdelete/SoftDeleteJobController.kt

```diff
@@ -50,13 +46,7 @@ class SoftDeleteJobController(
       ),
     ],
   )
-  @PostMapping("/delete-inactive-licences/{batchSize}")
+  @PostMapping("/delete-inactive-licences")
```

Comment:
removed this as its a bit easier to reason about only having it specified in one place